_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -0,0 +1,150 @@
+---
+name: Form field label is descriptive
+test_type: atomic
+
+description: |
+  This rule checks that labels describe the purpose of form field elements.
+
+success_criterion:
+- 2.4.6 # Headings and labels
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any HTML `label` element or other element referenced by `aria-labelledby` that is
+
+- either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
Editorial: 
Replace full stop with ", and" to get "- either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree), and" 
This will make it consistent with the list of the **Note:**